src/IonType.ts

```diff
@@ -15,29 +15,29 @@
 /** Ion value type enumeration class. */
 export class IonType {
```

Comment:
if you were so inclined, this class could be written as:
```javascript
export class IonType {
    constructor(
        /** The binary type ID for this Ion Type. */
        public readonly binaryTypeId: number,
        /** The textual name of this type. */
        public readonly name: string,
        ...
    }
}
```

Reply:
You bet I'm inclined! Done.

Reply:
Oh nice, I like this.